docs/next.config.mjs

```diff
@@ -7,6 +7,5 @@ const withNextra = nextra({
 });
 
 export default withNextra({
-  basePath: '/docs',
```

Comment:
we have this `basePath` for the reverse proxy on the website domain.

Reply:
we could take it out but we'd have to verify that it still works when you visit `mastra.ai/docs`

Reply:
There is a /docs folder now, so we don't need this anymore.